app/source/js/activities/serviceProfessionalService.js

```diff
@@ -258,6 +258,17 @@ function ViewModel(app) {
         
         return isin && hasPricing;
     }, this);
+
+    var baseNewServiceURL = this.newServiceURL;
+
+    this.newServiceURL = function(jobTitleID, pricingTypeID) {
+        if(this.client()) {
+            return '#!serviceProfessionalServiceEditor/' + jobTitleID + '/pricing_type/' + pricingTypeID + '/client' + this.clientID() + '/new';
```

Comment:
Regarding "/pricing_type/".
**Not** your fault, this just remember me that we must set a code convention for URLs, about to use camelCase, dash or underscore. Until now, as far I remember, we are using camelCase for the activity name (that is the first segment in the activity) and camelCase or dash in additional segments, but not underscore.

Is not a big deal, just a note, at some meeting we can discuss some convention to be consistent.